workspace.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@ericyoungson : I don't see why this file was checked in. This is probably related to your IDE? 🤔

Reply:
I have no idea. I only add files individually to the repo--hold on...

Reply:
Ok, rookie mistake. I've added `*code-worskpace` to my *~/.gitignore_global* file. _Shouldn't_ happen again! ;-)

Reply:
No worries. 😄